packages/sol-cov/src/artifact_adapters/0x.ts

```diff
@@ -0,0 +1,41 @@
+import * as fs from 'fs';
```

Comment:
I don't like the name of this file. Name it identical to the class it contains: `ZeroExArtifactAdapter` for grep-ability.